hs_core/page_processors.py

```diff
@@ -324,7 +327,8 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                'extended_metadata_layout': extended_metadata_layout,
                'bag_url': bag_url,
                'show_content_files': show_content_files,
```

Comment:
When ```show_content_files```  is False it implies the resource is 'discoverable' and 'private'. When ```show_content_files``` is True it implies either the resource is 'discoverable' and 'public' or resource is not 'discoverable' and 'private'.
```show_content_files``` by it self can't reflect the discoverable/public of the resource.